corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/controller.js

```diff
@@ -104,6 +104,34 @@ hqDefine("cloudcare/js/formplayer/menus/controller", function () {
         });
     };
 
+    const groupDisplays = function (displays, groupHeaders) {
+        const groupedDisplays = [];
+        let currentGroup = {
+            groupKey: null,
+            groupName: null,
+            displays: [],
+        };
+
+        displays.forEach(display => {
+            const groupKey = typeof display.get === 'function' ?  display.get('groupKey') : display['groupKey'];
```

Comment:
Not sure if I can do something better here. In one place the response object is passed in and in another a normal object.

Reply:
Does `_.get(object, path)` work here? I think underscore object methods often also work with things that are "object-like"

Reply:
getting this:
```
jQuery.Deferred exception: required is not defined anonymous@http://localhost:8000/static/underscore/underscore-min.js line 6 > Function:12:6
f@http://localhost:8000/static/underscore/underscore-min.js:6:15311
render@http://localhost:8000/static/marionette.templatecache/lib/marionette.templatecache.min.js:12:620
_renderTemplate@http://localhost:8000/static/backbone.marionette/lib/backbone.marionette.js:595:23
render@http://localhost:8000/static/backbone.marionette/lib/backbone.marionette.js:1884:12
```